packages/cli/test/integration/generators/app.integration.js

```diff
@@ -216,6 +216,45 @@ describe('app-generator with default values', () => {
   });
 });
 
+/** For testing if the app names with numbers are untouched */
+describe('app-generator with numbers in app name', () => {
+  const rootDir = path.join(__dirname, '../../../../../');
+  const defaultValProjPath = path.join(rootDir, 'sandbox/lb4-example');
+  const sandbox = path.join(rootDir, 'sandbox');
+  const pathToDefValApp = path.join(defaultValProjPath, 'lb4-example');
```

Comment:
so it means the project will be created inside 'sandbox/lb4-example/lb4-example'? Hmm why we have to use 2 lb4-example here?

Reply:
Following the pattern in https://github.com/strongloop/loopback-next/blob/bc2293f70ad4cd6c450b3413a1302f133245837d/packages/cli/test/integration/generators/app.integration.js#L183.